lib/PINNBench/DiffusionReaction/hpo.py

```diff
@@ -0,0 +1,116 @@
+import os
+import torch
+import numpy as np
+from pdebench.models.pinn.train import *
+from .model import FNN
+from deephyper.problem import HpProblem
+from deephyper.search.hps import CBO
+from deephyper.evaluator import profile, RunningJob
+from deephyper.stopper.integration import DeepXDEStopperCallback
+from deephyper.stopper import LCModelStopper
+from deephyper_benchmark.utils.json_utils import array_to_json
+from fvcore.nn import FlopCountAnalysis
+from deephyper_benchmark.integration.torch import count_params
+
+
+@profile
+def run(job: RunningJob) -> dict:
+    config = job.parameters
+    dataset = os.environ.get('DEEPHYPER_BENCHMARK_DATASET')
+
+    stopper_callback = DeepXDEStopperCallback(job)
+
+    val_loss, test_loss, losshistory, model = run_training(
+        net_class=FNN,
+        scenario="diff-react",
+        epochs=config["epochs"],
+        learning_rate=config["lr"],
+        model_update=500,
+        root_path="./DiffusionReaction/build/PDEBench-DH/pdebench/data/" + dataset,
```

Comment:
it would be safer to use an absolute path. 
```python
import os
DIR = os.path.dirname(os.path.abspath(__file__))
root_path = os.path.join(DIR, "build/PDEBench-DH/pdebench/data/" + dataset)
```